.buildkite/nightly-benchmarks/tests/descriptions.md

```diff
@@ -0,0 +1,67 @@
+
+## Latency tests
+
+This test suite aims to test vllm's end-to-end latency under a controlled setup.
+
+- Input length: 32 tokens.
+- Output length: 128 tokens.
+- Batch size: fixed (8).
+- Models: llama-3 8B, llama-3 70B, mixtral 8x7B.
+- Evaluation metrics: end-to-end latency (mean, median, p99).
+
+### Latency benchmarking results
+
+{latency_tests_markdown_table}
+
+## Throughput tests
+
+This test suite aims to test vllm's throughput.
+
+- Input length: randomly sample 200 prompts from ShareGPT dataset (with fixed random seed).
+- Output length: the corresponding output length of these 200 prompts.
+- Batch size: no constraint, so that vllm can batch as many requests as GPU memory permits.
```

Comment:
This is technically speaking not true: vLLM has default values for scheduling settings (such as `--max-num-seqs`, `--max-num-batched-tokens` etc) and utilizes continuous batching on top of those settings.

Reply:
Yes ur right. Maybe it is better to say 
"Batch size: dynamically determined by vllm to achieve maximum throughput."